### PR TITLE
Ss 558 create empty folders and gitkeep simplify local dev setup

### PR DESCRIPTION
**Pull request by sandstromviktor**
## Status

- [X] Ready
- [ ] Draft
- [ ] Hold

## Description

I have added empty folders with `.gitkeep` files as per Nikitas suggestion. I've also added these to the `.dockerignore` file. 

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [X] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [X] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [X] I have added a reviewer for this pull request
- [X] I have added myself as an author for this pull request

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+media
+charts/values
+sent_emails
```
Comment (churnikov) [inline .dockerignore:3]: Judging by ci build, these folders are not ignored by docker 🤔

**Comment by alfredeen** [comment]
Really nice work. The reason why the build and test workflow fails is that it contains a step that attempts to create a media folder, from before this change when this folder was missing from the repo.

mkdir: cannot create directory ‘media’: File exists

**Comment by churnikov** [comment]
> Really nice work. The reason why the build and test workflow fails is that it contains a step that attempts to create a media folder, from before this change when this folder was missing from the repo.
> 
> mkdir: cannot create directory ‘media’: File exists

Yes. But I thought that `.dockerignore` file file would fix this. It's seems like it is being ignored as it contains `media/` folder

**Comment by alfredeen** [comment]
> > Really nice work. The reason why the build and test workflow fails is that it contains a step that attempts to create a media folder, from before this change when this folder was missing from the repo.
> > mkdir: cannot create directory ‘media’: File exists
> 
> Yes. But I thought that `.dockerignore` file file would fix this. It's seems like it is being ignored as it contains `media/` folder. Or it's not and there is an issue with folder itself not being ignored.
> 
> Either way, it's not working :)

It could be that it is working correctly for docker (i.e. ignoring the folder). The github action is simply checking out the repo (with the media folder and all) and then attempting to create the media folder, which fails. If the media folder is pre-created and stored in the repo and probably the github action should not also create it.

**Comment by sandstromviktor** [comment]
Hej great feedback! Will fix this tomorrow!

**Pull request merged**